File: short_problems/Python/love_letter_mistery.py
#!/usr/local/bin/python

def find_min_operations(s):
    if len(s) == 0 or len(s) == 1:
        return 0
    inf, sup, count = 0, len(s) - 1, 0
    while inf < sup:
        count += abs(ord(s[inf]) - ord(s[sup]))
        inf += 1
        sup -= 1
    return count

if __name__ == "__main__":
    num_cases = int(input())
    for i in range(num_cases):
        s = str(input())
        print(find_min_operations(s))

File: short_problems/C++/love_letter_mistery.cpp
#include <iostream>
#include <string>
using namespace std;

inline int abs(char x) {
    return x < 0 ? -x : x;
}

int find_min_operations(const string &s)
{
    if (s.size() == 0 || s.size() == 1) {
        return 0;
    }
    auto inf = begin(s);
    auto sup = end(s) - 1;
    int count = 0;
    while (inf < sup) {
        count += abs(*inf - *sup);
        ++inf;
        --sup;
    }
    return count;
}

int main()
{
    int n;
    cin >> n;
    for (int i = 0; i < n; i++) {
        string s;
        cin >> s;
        cout << find_min_operations(s) << endl;
    }
    return 0;
}

